test/ProgressOnderwijsUtilsTests/XmlCompressionTest.cs

```diff
@@ -146,12 +146,13 @@ public void SaveToUtf8ExcludesIndentation()
     </nested>
 </test>");
 
-            var bytes = XmlCompression.ToUtf8(doc);
-            var str = UTF8.GetString(bytes);
+            var utf8BytesFromXml = XmlCompression.ToUtf8(doc);
+            var stringFromBytes = UTF8.GetString(utf8BytesFromXml);
 
-            PAssert.That(() => str == @"<test><nested><elements><here>
+            //XDocument.Parse/Serialize appears to sometimes lose CR's .net framework and core diverge here: we don't care.

```

Comment:
I believe it's just .NET Core on Linux.

Reply:
Zou best kunnen dat als je andersom met een cr-loze encoding deze file opslaat het dan op windows weer misgaat - ik dacht; we willen hier gewoon helemaal niet gevoelig voor zijn.

Reply:
Natuurlijk, maar er staat nu dat het een verschil tussen Core en Framework is, terwijl het een verschil is tussen de Windows en Linux-implementatie. (Niet dat er een Linux-implementatie is van Framework, maar ach.)

Reply:
sure, Ik weet dus gewoon niet of dat op windows beter is, want de file is nu toevallig met `\r`, en dat is zo'n stukje trivia waar je liever niet van afhankelijk wilt zijn.

Reply:
Als Unix nooit de \r gedropt had was de wereld een betere plek geweest.

Ik ben ook niet tegen de wijziging, maar het comment is misleidend.

Reply:
Oh ja hoor, het is weer linux' schuld